src/reynir_correct/config/GreynirCorrect.conf

```diff
@@ -3579,9 +3579,153 @@ Sjíti
 Súnníti
 evrópa
 sjálfstæðismaður
+samfylkingarmaður
+framsóknarmaður
+miðflokksmaður
+pírati
 Lenínismi
 Marxismi
 Darwinismi
+Heimsstyrjöld	# Fyrri/Seinni/Síðari Heimsstyrjöld
+Stríð 			# Kalda Stríðið
+Heimsveldi
+Bylting
+Uppreisn
+Meistaradeild
+Kreppa
+Endurreisn
+Jörð
+Tungl
+vesturlönd
+norðurlönd
+óskarsverðlaun 	#ekki leiðrétt, fer í 'if all(m.stofn not in stems for m in meanings)' en meanings = []
```

Comment:
"óskarsverðlaun" (lowercase) seems to be in BÍN, and is therefore not corrected.